CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contributing to the Online Purchasing Intention Predictor project
+
+This outlines how to propose a change to the Online Purchasing Intention Predictor project. Thank you so much for willing to contribute to it.
+
+### Pull request process
+  
+We recommend that you create a pull request by following steps:
+* Fork the repository and create your own branch.
+* Create a pull request and the teams will review your changes and approve them if the changes are considered meaningful.
+
+### Other document
+
+Team documents like meetings which are  not required by the project might be changed directly on main repository.
```

Comment:
Team documents not required by the project, such as meeting files, might be changed directly on **the** main repository.